templates/app.yaml

```diff
@@ -376,33 +369,6 @@ Resources:
             Resource:
               - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${OidcClientSecretKeyName}'
               - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${SsmParameterPrefix}*'
-  ResolveCustomerMarketplaceManagedPolicy:
-    Type: "AWS::IAM::ManagedPolicy"
-    Properties:
-      PolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Sid: ResolveCustomerMarketplace
-            Effect: Allow
-            Action: "aws-marketplace:ResolveCustomer"
-            Resource: "*"
-  DynamoRWManagedPolicy:
-    Type: AWS::IAM::ManagedPolicy
-    Properties:
-      PolicyDocument:
-        Version: '2012-10-17'
-        Statement:
-          - Sid: DynamoRW
-            Effect: 'Allow'
-            Action:
-              - 'dynamodb:*Get*'
-              - 'dynamodb:*List*'
-              - 'dynamodb:Query'
-              - 'dynamodb:*Write*'
-              - 'dynamodb:*Put*'
-            Resource:
-              - !ImportValue
-                'Fn::Sub': '${AWS::Region}-marketplace-dynamo-MarketplaceDynamoDBTableArn'
```

Comment:
We created a dynamo db in AWS SC accounts to support this functionality.  should we remove it as well?  here's where we created the db https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/scipool/config/prod/marketplace-dynamo.yaml

Reply:
Ya, I already made the necessary PR (with a dpulls dependency on this PR): https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/207/files